tests/foreman/cli/test_repository_set.py

```diff
@@ -147,3 +149,118 @@ def test_repositoryset_available_repositories(self):
+                enabled = repo['enabled']
+        self.assertEqual(enabled, 'true')
+
+    def test_repositoryset_enable_by_label(self):
+        """@Test: Enable repo from reposet by org label, reposet and product
+        names
+
+        @Feature: Repository-set
+
+        @Assert: Repository was enabled
+
+        """
+        org = make_org()
+        manifest = manifests.clone()
+        upload_file(manifest, remote_file=manifest)
+        result = Subscription.upload({
+            u'file': manifest,
+            u'organization-id': org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = RepositorySet.enable({
+            u'name': REPOSET['rhva6'],
+            u'organization-label': org['label'],
+            u'product': PRDS['rhel'],
+            u'releasever': '6Server',
+            u'basearch': 'x86_64',
+        })
+        self.assertEqual(result.return_code, 0)
+        result = RepositorySet.available_repositories({
+            u'name': REPOSET['rhva6'],
+            u'organization-label': org['label'],
+            u'product': PRDS['rhel'],
+        })
+        self.assertEqual(result.return_code, 0)
+        for repo in result.stdout:
+            if repo['release'] == '6Server' and repo['arch'] == 'x86_64':
+                enabled = repo['enabled']
+        self.assertEqual(enabled, 'true')
+
+    def test_repositoryset_enable_by_id(self):
+        """@Test: Enable repo from reposet by IDs of reposet, org and product
+
+        @Feature: Repository-set
+
+        @Assert: Repository was enabled
+
+        """
+        org = make_org()
+        manifest = manifests.clone()
+        upload_file(manifest, remote_file=manifest)
+        result = Subscription.upload({
+            u'file': manifest,
+            u'organization-id': org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        product_id = Product.info({
+            u'name': PRDS['rhel'],
+            u'organization-id': org['id'],
+        }).stdout['id']
+        reposet_id = RepositorySet.info({
+            u'name': REPOSET['rhva6'],
+            u'organization-id': org['id'],
+            u'product-id': product_id,
+        }).stdout['id']
+        result = RepositorySet.enable({
+            u'id': reposet_id,
+            u'organization-id': org['id'],
+            u'product-id': product_id,
+            u'releasever': '6Server',
+            u'basearch': 'x86_64',
+        })
+        self.assertEqual(result.return_code, 0)
+        result = RepositorySet.available_repositories({
+            u'id': reposet_id,
+            u'organization-id': org['id'],
+            u'product-id': product_id,
+        })
+        self.assertEqual(result.return_code, 0)
+        for repo in result.stdout:
+            if repo['release'] == '6Server' and repo['arch'] == 'x86_64':
+                enabled = repo['enabled']
+        self.assertEqual(enabled, 'true')
```

Comment:
This variable will be unset if no repositories satisfy the conditional directly above.